src/store/sagas/login.ts

```diff
@@ -244,6 +244,7 @@ async function downgradeToPleb() {
   dbManager.updateObjectById(RealmSchema.KeeperApp, app.id, {
     subscription: updatedSubscription,
   });
+  store.dispatch(setSubscription(updatedSubscription.name));
```

Comment:
This will permanently put the user on pleb even when internet connection returns no?